package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "vote_tracker",
+  "version": "0.1.0",
+  "description": "an app that tracks which kitty shall live",
+  "main": "index.js",
```

Comment:
Shouldn't this point to the name of the .js file that you want to run with node (in this case, server.js)? I'm sure you fixed this by the time you deployed to heroku.